File: quests/freportw/Hemia_Skemner.pl
############################################
# ZONE: West Freeport (freportw)
# DATABASE: PEQ-Velious
# LAST EDIT DATE: May 1,2005
# VERSION: 1.0
# DEVELOPER: MWMDRAGON
#
# *** NPC INFORMATION ***
#
# NAME: Hemia_Skemner
# ID: 9071
# TYPE: Wizard Guild Master
# RACE: Human
# LEVEL: 61
#
# *** ITEMS GIVEN OR TAKEN ***
#
# A Locked Book ID-13863
#
#
# *** QUESTS INVOLVED IN ***
#
#1 - Lydl Mastat
#
# *** QUESTS AVAILABLE TO ***
#
#1 - All
#
# *** NPC NOTES ***
#
#
############################################

sub EVENT_SAY
{
if($text=~/Hail/i)
 {
 quest::say("How are you? Welcome to the Academy of Arcane Science. Please feel free to roam. May knowledge be your goal. Be you a [fellow scientist] or visitor, I am sure you will find whatever you seek.");
 }

 if($text=~/fellow scientist/i)
 {
 quest::say("That is good. I hope you can handle the knowledge of arcane science. Some have gone mad conducting research. Such a person is [Lydl Mastat].");
 }

 if($text=~/Lydl Mastat/i)
 {
 quest::say("Lydl Mastat is nothing more than a young apprentice of mine. He went mad working on a formula for deep water breathing. He now roams the outskirts of Freeport boasting of power he does not have. Someone should [teach Lydl a lesson].");
 }

 if($text=~/teach Lydl a lesson/i)
 {
 quest::say("Then go. He appears near the gates of Freeport from time to time. He is not powerful, but he has a secret. Nothing save magic can harm him. This was the result of an experimental potion he drank. Slay him and return his locked spellbook to me.");
 }

}

sub EVENT_ITEM
{
   # A Locked Book ID-13863
   if($itemcount{13863} == 1)
   {
   quest::say("You have done the world of magic a great justice, $name.");
   quest::ding();
   quest::givecash ("9","9","9","0");

   # Arcane Scientists Faction
   quest::faction("11","1");
   # Knights of Truth Faction
   quest::faction("184","1");
   # Opal Dark Briar Faction
   quest::faction("235","-1");
   # Freeport Militia Faction
   quest::faction("105","-1");
   }
}


#END of FILE Zone:freportw  ID:9071 -- Hemia_Skemner

File: freportw/Hemia_Skemner.lua
-- items: 13863
function event_say(e)
	if(e.message:findi("hail")) then
		e.self:Say("How are you? Welcome to the Academy of Arcane Science. Please feel free to roam. May knowledge be your goal. Be you a [fellow scientist] or visitor, I am sure you will find whatever you seek.");
	elseif(e.message:findi("fellow scientist")) then
		e.self:Say("That is good. I hope you can handle the knowledge of arcane science. Some have gone mad conducting research. Such a person is [Lydl Mastat].");
	elseif(e.message:findi("lydl mastat")) then
		e.self:Say("Lydl Mastat is nothing more than a young apprentice of mine. He went mad working on a formula for deep water breathing. He now roams the outskirts of Freeport boasting of power he does not have. Someone should [teach Lydl a lesson].");
	elseif(e.message:findi("teach lydl a lesson")) then
		e.self:Say("Then go. He appears near the gates of Freeport from time to time. He is not powerful, but he has a secret. Nothing save magic can harm him. This was the result of an experimental potion he drank. Slay him and return his locked spellbook to me.");
	end
end

function event_trade(e)
	local item_lib = require("items");

	if(item_lib.check_turn_in(e.trade, {item1 = 13863})) then -- A Locked Book
		e.self:Say(string.format("You have done the world of magic a great justice, %s.",e.other:GetName()));
		e.other:Ding();
		e.other:Faction(220,10,0); -- Arcane Scientists
		e.other:Faction(281,2,0); -- Knights of Truth
		e.other:Faction(296,-1,0); -- Opal Dark Briar
		e.other:Faction(330,-1,0); -- Freeport Militia
		e.other:AddEXP(500);
		e.other:GiveCash(9,9,5,0);
	end
	item_lib.return_items(e.self, e.other, e.trade)
end

-- END of FILE Zone:freportw -- Hemia_Skemner
